scripts/qa_crowbarsetup.sh

```diff
@@ -563,6 +563,12 @@ function get_unclustered_sles12plus_nodes
     echo $sles12plusnodes
 }
 
+function get_non_controller_nodes
+{
+    sles12nodes=get_unclustered_sles12plus_nodes
+    controller_node=get_novacontroller
```

Comment:
I don't think this will work for two reasons:

- when using HA, what you get here is not what you expect :-)
- more importantly, when you call this function, the nova proposal is being written by the script, so you will actually get the nova controller from the proposal when it was created, not the nova controller from the proposal that you are writing, which can be different. See the `proposal_set_value nova default "['deployment']['nova']['elements']['${role_prefix}-controller']" "['cluster:$clusternameservices']"` line, for instance.